### PR TITLE
issue#1846

### LINKED ISSUE #1846: Collections: New/Edit form should include collection type in page title
### Descriptive summary

New/Edit form should identify the collection type in the page header.

### Rationale

Provide additional information to user about what they are creating/editing. 

### Expected behavior

New collection title should be: `Create New <collection_type.title>`  Ex. `New Exhibit`

Edit collection title should be: `Edit <collection_type.title>: <collection.title>`  Ex. `Edit User Collection: My Favorites`

### Actual behavior

New collection title is: `Create New Collection`

Edit collection title is: `Edit Collection: <collection.title>`

### Steps to reproduce the behavior

1. Create a collection of type User Collection

- [x] Verify the title at the top of the new form is: `Create New User Collection`

1. Set title to 'My Favorites' and Save

- [x] Verify the title at the top of the edit form is: `Edit User Collection: My Favorites`

### Related work

#1374 Modify existing new/edit view to include partials for new tabs

### PR DESCRIPTION
**Pull request by gordonleacock**
Fixes #1846

Add the collection type in the collection EDIT and NEW pages.

@samvera/hyrax-code-reviewers

### CODE REVIEW

File: app/helpers/hyrax/title_helper.rb [hunk 1]
```diff
@@ -14,6 +14,13 @@ def curation_concern_page_title(curation_concern)
       construct_page_title("New #{curation_concern.human_readable_type}")
     end
   end
+  
+  def collection_type_title(coll)
+    this_coll_index = coll.collection_type.id - 1
```
Comment (elrayle) [inline app/helpers/hyrax/title_helper.rb]: With the suggestions that follow, this won't be needed.

File: app/views/hyrax/dashboard/collections/edit.html.erb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <% provide :page_title, construct_page_title( t('.header', title: @form.title.first) ) %>
 
-<%= provide :page_header do %>
-  <h1><span class="fa fa-edit"></span><%= t('.header', title: @form.title.first) %></h1>
+<% provide :page_header do %>
+  <h1><span class="fa fa-edit"></span>EDIT <%= collection_type_title(@collection) %>: <%= @form.title.first %></h1>
```
Comment (elrayle) [inline app/views/hyrax/dashboard/collections/edit.html.erb]: I wonder if you can use the following in place of this code...
```
<% provide :page_title, construct_page_title( t('.header', type_title: @collection.collection_type.title, title: @form.title.first) ) %>
<% provide :page_header do %>
  <h1><span class="fa fa-edit"></span><%= t('.header', type_title: @collection.collection_type.title, title: @form.title.first) %></h1>
<% end %>
```
```

File: app/views/hyrax/dashboard/collections/new.html.erb [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <% provide :page_title, construct_page_title( t('.header') ) %>
 
 <% provide :page_header do %>
-  <h1><span class="fa fa-edit"></span> <%= t('.header') %></h1>
+  <h1><span class="fa fa-edit"></span> <%= t('.header') %> <%= collection_type_title(@collection) %> </h1>
```
Comment (elrayle) [inline app/views/hyrax/dashboard/collections/new.html.erb]: I wonder if you can use the following in place of this code...
```
<% provide :page_title, construct_page_title( t('.header', type_title: @collection.collection_type.title) ) %>
<% provide :page_header do %>
  <h1><span class="fa fa-edit"></span> <%= t('.header', type_title: @collection.collection_type.title) %> </h1>
<% end %>
```

File: config/locales/hyrax.en.yml [hunk 4]
```diff
@@ -492,7 +492,7 @@ en:
         admin:                  "Administration"
       collections:
         edit:
-          header: "Edit Collection: %{title}"
+          header: "%{title}"
```
Comment (elrayle) [inline config/locales/hyrax.en.yml]: Then this becomes...
```
     header: "Edit %{type_title}: %{title}"
```

File: config/locales/hyrax.en.yml [hunk 5]
```diff
@@ -580,7 +580,7 @@ en:
           this_collection_in_other_collections_description: "You can add this collection to other collections as a sub-collection."
 
         new:
-          header: Create New Collection
+          header: Create New
```
Comment (elrayle) [inline config/locales/hyrax.en.yml]: And this become...
```
     header: "New %{type_title}"
```

**Comment by jcoyne** [comment]
Please change the commit message to something that describes the work you’ve done.

**Pull request merged**